internal/trigger/http/rest.go

```diff
@@ -69,6 +69,8 @@ func (trigger *Trigger) requestHandler(writer http.ResponseWriter, r *http.Reque
 		CorrelationID: correlationID,
 	}
 
+	trigger.logging.Trace(fmt.Sprintf("Received message from http. Corrilation Id is '%s'", correlationID))
```

Comment:
Corrilation should be "Correlation"

Reply:
You can set the correlation-id as a key/value in the logging statement per this example
https://github.com/edgexfoundry/edgex-go/blob/41176abe2c12f7272535b4ba75395318ca549926/internal/pkg/correlation/middleware.go#L34